app/src/lib/open-shell.ts

```diff
@@ -15,14 +15,20 @@ export function openShell(fullPath: string, shell?: string) {
     return spawn('START', [shell || 'cmd'], { shell: true, cwd: fullPath })
   }
 
+  if (__LINUX__) {
+    const commandArgs = ['--working-directory', fullPath]
+    return spawn('gnome-terminal', commandArgs, { shell: true })
+  }
+
   return fatalError('Unsupported OS')
 }
 
 export function isGitOnPath(): Promise<boolean> {
   // Modern versions of macOS ship with a Git shim that guides you through
   // the process of setting everything up. We trust this is available, so
   // don't worry about looking for it here.
-  if (__DARWIN__) {
+  // I decide linux user have git too :)
```

Comment:
Looks like this need to change. Anyway, where is the function being used. I know desktop bundled git binary...